tests/readConcern/readconcern-serialization-001.phpt

```diff
@@ -38,57 +34,13 @@ object(MongoDB\Driver\ReadConcern)#%d (%d) {
 
 object(MongoDB\Driver\ReadConcern)#%d (%d) {
   ["level"]=>
-  string(12) "linearizable"
+  string(12) "local"
 }
 bool(true)
-C:26:"MongoDB\Driver\ReadConcern":38:{a:1:{s:5:"level";s:12:"linearizable";}}
+C:26:"MongoDB\Driver\ReadConcern":38:{a:1:{s:5:"level";s:12:"setTestLevel";}}
```

Comment:
This test is failing in [Evergreen](https://spruce.mongodb.com/version/63223ccc3e8e865cf15548a1/tasks).